v1/src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -127,9 +127,7 @@ public interface SplunkOptions extends PipelineOptions {
         optional = true,
         description = "Disable SSL certificate validation.",
         helpText =
-            "Disable SSL certificate validation (true/false). Default false (validation "
-                + "enabled). If true, the certificates are not validated (all certificates are "
-                + "trusted) and  `rootCaCertificatePath` parameter is ignored.")
+            "Whether to disable SSL certificate validation. When set to false, validation is enabled. When set to true, the certificates are not validated, all certificates are trusted, and the `rootCaCertificatePath` parameter is ignored.")
```

Comment:
What's the default?

Reply:
https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1490/commits/3276300d71c8df1217d2a48984c566ea72af8178 - corrected. Defaults to false.